src/screens/send/components/header-step-by-step/header-step-by-step.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { View, Dimensions, TouchableOpacity } from 'react-native';
+import stylesProvider from './styles';
+import { withTheme, IThemeProps } from '../../../../core/theme/with-theme';
+import { Text } from '../../../../library';
+import { smartConnect } from '../../../../core/utils/smart-connect';
+import { BASE_DIMENSION } from '../../../../styles/dimensions';
+
+export interface IExternalProps {
+    steps: { title: string; selected?: boolean }[];
+    selectPreviousStep: (index: number) => void;
```

Comment:
too complicated with selectPreviousStep
Make it easier. pass a list of steps with selected 0 or 1 - one step will be selected
Example:
[0, 0, 0, 1, 0] - mark all steps (visually) as selected until you reach the 1 value, all after 1 are not selectee